src/olympia/users/tests/test_models.py

```diff
@@ -380,6 +380,53 @@ def test_empty_email(self):
         assert exc_info.value.messages[0] == 'This field is required.'
 
 
+class TestOnChangeName(TestCase):
+    def setUp(self):
+        super(TestOnChangeName, self).setUp()
+
+        # We're in a regular TestCase class so index_addons should have been
+        # mocked.
+        from olympia.addons.tasks import index_addons
+        self.index_addons_mock = index_addons
```

Comment:
I might be blind but where is this being set? I can only find it being used in src/olympia/addons/tests/test_commands.py

Reply:
The mock? It comes from `start_es_mocks()`, which is called as a fixture in `conftest.py`.

Reply:
eh, nevermind, I was simply blind earlier